src/test/java/com/example/digitalmenuapi/controllerTest/menuControllerTest.java

```diff
@@ -0,0 +1,73 @@
+package com.example.digitalmenuapi.controllerTest;
+
+import com.example.digitalmenuapi.controller.MenuItemsController;
+import com.example.digitalmenuapi.controller.UserMenuController;
+import com.example.digitalmenuapi.model.MenuItems;
+import com.example.digitalmenuapi.service.MenuItemsService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Assert;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class menuControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Mock
+    private MenuItemsService menuItemsService;
+
+    @InjectMocks
+    private MenuItemsController menuItemsController;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @BeforeEach
+    public void setUp() {
+        mockMvc = MockMvcBuilders.standaloneSetup(menuItemsController).build();
+    }
+
+    @Test
+    public void createNewSandwichesTest() throws Exception{
```

Comment:
You could create tests that take an empty body or invalid values to test for error handling